tests/test_worker.py

```diff
@@ -14,19 +13,18 @@
 # GNU General Public License for more details.
 #
 # You should have received a copy of the GNU General Public License
-# along with this program; if not, write to the Free Software
```

Comment:
Why this part has been removed ?

Reply:
I've updated the license to the latest format: https://www.gnu.org/licenses/gpl-3.0.en.html. See **How to Apply These Terms to Your New Programs** section.

I'll update license format progressively, cause it's annoying do it at the same time.

Reply:
Ok, thanks for the clarification